react-debugging.html

```diff
@@ -63,22 +63,29 @@
           // No need to change anything ABOVE this line ^
 
           return(
+            // added opening and closing fragments
+            // I move the div segment towards the top
+            // fixed the buttons onClick
+            <> 
             <h1>Draw a Card</h1>
-            <button onClick={}>
-              Click to Draw a Card
-            </button>
-            <button onClick={shuffle}>
-              Return Cards and Shuffle Deck
-            </button>
             <div style={handStyles}>
```

Comment:
Interesting. I am curious of what you saw on the DOM. The goal of this challenge was to allow you to follow the prompts in the console. Good job on finding a way to get an output. Best practice is to follow the error messages from the console.